common/lib/xmodule/xmodule/course_module.py

```diff
@@ -662,7 +662,14 @@ class CourseFields(object):
         # Ensure that courses imported from XML keep their image
         default="images_course_image.jpg"
     )
-
+    issue_badges = Boolean(
+        display_name=_("Issue OpenBadges"),
+        help=_(
+            "Issue OpenBadges badges for this course. Badges are generated when certificates are created."
+        ),
+        scope=Scope.settings,
+        default=False
```

Comment:
We talked of keeping this option as a way to deactivate it for some courses in the specs, but to put it on by default.